src/app/pages/news/12.0/04-engine-cockpit.md

```diff
@@ -1,14 +1,14 @@
 ## Engine Cockpit {#engineCockpit}
 
-The engine cockpit has been improved with many small features that makes the lives of operators easier. 
+The engine cockpit has been updated with new features designed to simplify operations. 
 
-- **Migration Wizard**: The completely overhauled migration wizard supports migrating an Engine from one version to another, be it a LTS to next LTS migration or a migration from one update release to another.
-- **Health Check**: The new Engine health check service analyzes with a lot of different checks the health of your Engine and warns if your Engine is in an unhealthy situation.
-- **SSL / TLS Tester**: Manage your keys and certificates stored in the key and trust store in the new SSL view. Use the TLS tester to test connections to databases, REST, and SOAP services, and directly import missing certificates.
-- **Monitoring**: Analyze what is happening inside your Engine by using eleven new monitor views: System Database Info, Class Histogram, Threads, Flight Recorder, Notifications, Session, Jobs, Health, Documents, Start Events, and Intermediate Events.
-- **Security Export**: Export all users, roles, the roles of users, and permissions to Excel files for audits or documentation.
-- **Properties and Features**: Edit the properties or features of REST clients or SOAP web services.
-- **More Improvements**: Active Directory and Entra ID browsers, better configuration file editor, improved search engine view, user substitutes and absences management, merging and changing the security system of an application.
+- **Migration Wizard**: The migration wizard has been redesigned to assist with migrating an Engine between versions, including upgrades from one LTS (Long-Term Support) version to the next or between update releases.
+- **Health Check**: The new health check service performs various checks to analyze the Engine's health, providing warnings if potential issues are detected.
+- **SSL / TLS Tester**: Keys and certificates can now be managed through the SSL view. The TLS tester helps verify connections to databases, REST, and SOAP services, with the option to import missing certificates.
+- **Monitoring**: The monitoring capabilities now include eleven new views, such as System Database Info, Class Histogram, Threads, Flight Recorder, Notifications, Session, Jobs, Health, Documents, Start Events, and Intermediate Events, for better operational insights.
+- **Security Export**: Users, roles, user-role assignments, and permissions can now be exported to Excel files to facilitate audits or documentation.
+- **Properties and Features**: The properties and features of REST clients or SOAP web services can be edited directly from the cockpit.
+- **More Improvements**: Enhanced Active Directory and Entra ID browsers, improved configuration file editor, updated search engine view, user substitute and absence management, and options for merging or modifying application security.
```

Comment:
Here 'application security' is not the same as the security system of an application. In Ivy you have 'Security Systems' and 'Applications'.
What the features do is:
- You can merge two security systems (users, roles of both security systems are taken over. If a user or role exists in both sec sys they are merged)
- You can change which security system an application is assigned too.

Reply:
So then I would change it to that: "... , and options for merging and changing the security system of an application."
What do you think?